src/Microsoft.AspNet.OData.Shared/Builder/EdmTypeBuilder.cs

```diff
@@ -29,6 +29,7 @@ internal class EdmTypeBuilder
         private readonly Dictionary<IEdmStructuredType, ModelBoundQuerySettings> _structuredTypeQuerySettings = new Dictionary<IEdmStructuredType, ModelBoundQuerySettings>();
         private readonly Dictionary<Enum, IEdmEnumMember> _members = new Dictionary<Enum, IEdmEnumMember>();
         private readonly Dictionary<IEdmStructuredType, PropertyInfo> _openTypes = new Dictionary<IEdmStructuredType, PropertyInfo>();
+        private readonly Dictionary<IEdmStructuredType, PropertyInfo> _instanceAnnotations = new Dictionary<IEdmStructuredType, PropertyInfo>();
```

Comment:
>_instanceAnnotations [](start = 70, length = 20)

This really isn't a dictionary of instance annotations, it's a dictionary of types that support instance annotations, right?  Maybe rename something like "_instanceAnnotatableTypes"?

Reply:
#Resolved